imports/plugins/included/product-variant/client/templates/products/productDetail/variants/variantForm/variantForm.js

```diff
@@ -114,6 +133,74 @@ Template.variantForm.helpers({
         });
       };
     };
+  },
+  checkIfProviderIsEnabled: function () {
```

Comment:
this function should just be named `isProviderEnabled` since it returns a boolean